### PR TITLE
Update nao motor limits.

### PR DESCRIPTION
**Pull request by DavidMansolino**
Dual PR omichel/Webots#7628  

Preview: https://www.cyberbotics.com/doc/guide/nao?version=update-nao-motor-limits

### CODE REVIEW

**Comment by DavidMansolino** [comment]
:+1: 

(hide whitespace diff for a simpler review :wink: )

File: guide/scenes/nao/nao.meta.json [hunk 1]
```diff
@@ -1,730 +1,730 @@
...
-      "type": "RotationalMotor", 
-      "minPosition": -0.388685, 
+    },
+    {
+      "transformID": "n10153",
+      "name": "RAnkleRoll",
+      "maxPosition": 0.388685,
+      "position": 0,
+      "type": "RotationalMotor",
+      "minPosition": -0.785922,
```
Comment (omichel) [inline guide/scenes/nao/nao.meta.json]: The value in the NAO doc is -0.768992

File: guide/scenes/nao/nao.meta.json [hunk 2]
```diff
@@ -1,730 +1,730 @@
...
-      "name": "RAnkleRoll", 
-      "maxPosition": 0.785922, 
-      "position": 0, 
-      "type": "RotationalMotor", 
-      "minPosition": -0.388685, 
+    },
+    {
+      "transformID": "n10153",
+      "name": "RAnkleRoll",
+      "maxPosition": 0.388685,
```
Comment (omichel) [inline guide/scenes/nao/nao.meta.json]: The value in the NAO doc is 0.397935

File: guide/scenes/nao/nao.meta.json [hunk 3]
```diff
@@ -1,730 +1,730 @@
...
-      "type": "RotationalMotor", 
-      "minPosition": -0.768992, 
+    },
+    {
+      "transformID": "n10285",
+      "name": "LAnkleRoll",
+      "maxPosition": 0.768992,
+      "position": 0,
+      "type": "RotationalMotor",
+      "minPosition": -0.397761,
```
Comment (omichel) [inline guide/scenes/nao/nao.meta.json]: The value in the NAO doc is -0.397880

File: guide/scenes/nao/nao.meta.json [hunk 4]
```diff
@@ -1,730 +1,730 @@
...
-      "name": "LAnkleRoll", 
-      "maxPosition": 0.397761, 
-      "position": 0, 
-      "type": "RotationalMotor", 
-      "minPosition": -0.768992, 
+    },
+    {
+      "transformID": "n10285",
+      "name": "LAnkleRoll",
+      "maxPosition": 0.768992,
```
Comment (omichel) [inline guide/scenes/nao/nao.meta.json]: The value in the NAO doc is 0.769001

**Comment by DavidMansolino** [comment]
The difference is due to the fact that the values are computed using the limits in degree, but I will update them to use directly the limits described in radians in the doc (not sure which one is the most correct, but in any case, the value in radians is more precise).

**Pull request merged**